srcpkgs/shortwave/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'shortwave'
+pkgname=shortwave
+version=2.0.1
+revision=1
+_short_rev="9bfc5ee"
+build_style=meson
+build_helper=rust
+configure_args=" -Dtag=${_short_rev}"
```

Comment:
`$short_rev` is used only once, so it can be removed, but what is the value of tag? Do we need it?

Reply:
I think this removed a hostmakedepend on git, I'll just add the dependency and be done with it.